### PR TITLE
New version: GitHub.GitHubDesktop.Beta version 3.4.2-beta3

### LINKED ISSUE #159348: [Package Issue]: Package.Identifier GitHub Desktop
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

No applicable update found.

### Brief description of your issue

GitHub Desktop doesn't keep it up to date on the winget platform

### Steps to reproduce

winget search --name "GitHub"

### Actual behavior

image

### Expected behavior

image

### Environment

```raw
[winget - Normal]
[Powershell 7.5.0 - preview.3]
[Windows 11 24H2 Dev channel June 2024]
```

### Screenshots and Logs

_No response_

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This PR resolves #159348

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159350)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 886.)

File: manifests/g/GitHub/GitHubDesktop/Beta/3.4.2-beta3/GitHub.GitHubDesktop.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://desktop.githubusercontent.com/github-desktop/releases/3.4.2-beta3-6edaffed/GitHubDesktopSetup-x64.msi
+  InstallerSha256: 0FAF2FD887F72A441E7F9B912A3C681ACCED8FE67E0CED8DF2D43CD12C5C8BEE
+  ProductCode: '{8ABF5209-DD0D-497F-91AF-4C6293B3222C}'
+  AppsAndFeaturesEntries:
+  - DisplayName: GitHub Desktop Deployment Tool
+    DisplayVersion: 3.4.2.0
+    UpgradeCode: '{00D8E2EE-13EA-5BEB-87F0-70EFC46A7D4A}'
```
Comment (Dragon1573) [inline manifests/g/GitHub/GitHubDesktop/Beta/3.4.2-beta3/GitHub.GitHubDesktop.Beta.installer.yaml:33]:
```suggestion
```

Should be removed according to https://github.com/microsoft/winget-pkgs/pull/159350#issuecomment-2186908992

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**